app/models/family.rb

```diff
@@ -938,6 +938,39 @@ def actual_enrollments_number
         }
       ]).first.try(:[], "hbx_enrollments") || 0
     end
+
+    def actual_enrollments_number_by_year(year)
+      collection.aggregate([
+                             {
+                               "$unwind": "$households"
+                             },
+                             {
+                               "$unwind": "$households.hbx_enrollments"
+                             },
+                             {
+                               "$match": {
+                                 "households.hbx_enrollments.aasm_state": {
+                                   "$nin": [HbxEnrollment::WAIVED_STATUSES, HbxEnrollment::CANCELED_STATUSES, 'shopping']
```

Comment:
i wonder if at some point we want a new constant containing all these statuses and giving the collection a semantically meaningful name. (this doesn't have to be that point :) )